src/styles/foundation/spacing.scss

```diff
@@ -1,12 +1,20 @@
+@import '../polaris-tokens/spacing.map';
+
 $unit: 4px;
 
-$spacing-data: (
-  none: 0,
-  extra-tight: rem($unit),
-  tight: rem($unit * 2),
-  base: rem($unit * 4),
-  loose: rem($unit * 5),
-  extra-loose: rem($unit * 8),
+$spacing-data: $polaris-spacing-map;
```

Comment:
This means spacing-data ends up containing both `spacing-none` and `none`, `spacing-extra-tight` and `extra-tight` etc.

That duplication feels quite odd to me, I think we should keep to a single name for each size.